arm-apimanagement/2016-07-07/swagger/apimdeployment.json

```diff
@@ -0,0 +1,1236 @@
+          },
+          {
+            "$ref": "#/parameters/ServiceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully got the ApiManagement Service Resource.",
+            "schema": {
+              "$ref": "#/definitions/ApiServiceResource"
+            }
+          },
+          "404": {
+            "description": "ApiManagement Service not found.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "ResourceProvider"
+        ],
+        "operationId": "ApiManagementService_Delete",
+        "description": "Deletes existing Api Management service.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ServiceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+           "200": {
+            "description": "Service was successfully deleted."
+          },
+          "204": {
+            "description": "Service is already deleted."
+          },
+          "404": {
+            "description": "ApiManagement Service not found.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ApiManagement/service/": {
+      "get": {
+        "tags": [
+          "ResourceProvider"
+        ],
+        "operationId": "ApiManagementService_ListByResourceGroup",
+        "description": "List all Api Management services within a resource group.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Return the Api Service List.",
+            "schema": {
+              "$ref": "#/definitions/ApiServiceListResult"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+	  "/subscriptions/{subscriptionId}/providers/Microsoft.ApiManagement/service/": {
+     "get": {
+        "tags": [
+          "ResourceProvider"
+        ],
+        "operationId": "ApiManagementService_ListBySubscriptionId",
```

Comment:
ApiManagementService_List is the convention. Please read all the recommended patterns https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-good-patterns.md